### ISSUE TITLE
Questions about the loss function

### ISSUE THREAD

**Issue by dawniset** [dfloryan/neural-manifold-dynamics#1]
Thanks for your great contribution! I have some doubts about the training logic for autoencoder. In paper the loss function autoencoder is related to the evolution, however in the code, it's a simple reconstruction task, the evolution part and autoencoder part is seperated. Does the dynamics module have other functions besides playing a role in the subsequent evolution of the dynamics trajectory?

**Comment by dfloryan** [comment]
The autoencoders are trained first. No information about the dynamics is used when training the autoencoders. Then, maps for the local dynamics are trained. This second phase uses the already trained autoencoders since the dynamics are learned in the local coordinates of each chart. 

The loss function (Eq. 4 in the paper) is for the second phase. In that loss function, $\phi_\alpha$ is the already trained encoder for chart $\alpha$, and it is not modified during this phase. Only $G_\alpha$ (the dynamics map in the local coordinates of chart $\alpha$) is modified during this training phase.

**Comment by dawniset** [comment]
Thank you for your kind explanation！
